packages/ui/.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Just wondering what this file is doing in here?

Reply:
Hi,
This file was added, to prevent auto generated files e.g `storybook-static/**` from being committed as part of the tracked changes.

Reply:
Hi @MildTomato, the unrequired lines has been removed. Kindly take a look.